src/index.ts

```diff
@@ -149,7 +149,15 @@ export function register (options: Options = {}): () => Register {
   const shouldCache = !!(options.cache == null ? DEFAULTS.cache : options.cache)
   const fast = !!(options.fast == null ? DEFAULTS.fast : options.fast)
   const project = options.project || DEFAULTS.project
-  const cacheDirectory = options.cacheDirectory || DEFAULTS.cacheDirectory || join(tmpdir(), 'ts-node')
+
+  function getTmpDir(): string {
```

Comment:
No worries, it seems reasonable. I just wanted to find out why, thanks! 👍  There is a `userInfo` method, but it appears to be new in v6 so won't be very useful to us here.